src/scenarios/shared/startup.py

```diff
@@ -78,7 +78,8 @@ def runtests(self, apptorun: str, **kwargs):
         RunCommand(startup_args, verbose=True).run()
 
 
-        if runninginlab() and uploadtokenpresent():
-            import upload
+        if runninginlab():
```

Comment:
@DrewScoggins in particular this seem okay to you? I think this is closer to the behavior I'd like.

Reply:
Yeah, this seems better. We always want to save artifacts, but only do the upload if we are in an internal run.